ast/Earthfile

```diff
@@ -23,15 +23,4 @@ unit-test:
 
 # generate (re-)generates generated code in this module.
 generate:
-    BUILD +mocks
     BUILD ./parser+parser
-
-# mocks runs 'go generate' against this module and saves generated mock files
-# locally.
-mocks:
-    FROM +code
-    RUN go install git.sr.ht/~nelsam/hel/v4@latest && go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
This target failed due to some problem with the upstream `git.sr.ht/~nelsam/hel` and doesn't appear to be needed anyways (no `helmheim` files in `ast/`)